src/lambda/wizards/samDeployWizard.ts

```diff
@@ -117,18 +240,53 @@ export class SamDeployWizard extends MultiStepWizard<SamDeployWizardResponse> {
 class SamTemplateQuickPickItem implements vscode.QuickPickItem {
     public readonly label: string
 
-    public readonly description: string
+    public description?: string
+    public detail?: string
+
+    private readonly workspacePath?: string
 
     public constructor(
         public readonly uri: vscode.Uri
     ) {
-        this.label = path.basename(uri.fsPath)
-        this.description = path.dirname(uri.fsPath)
+        let label: string
+        const workspace = vscode.workspace.getWorkspaceFolder(uri)
```

Comment:
Will `SamTemplateQuickPickItem` ever need to be mocked? If so, make `getWorkspaceFolder` injectable.

Reply:
Also, I'm nitpicking here, but a workspace and a workspace folder are not the same thing--suggest to change the identifiers/comments accordingly.

Reply:
Tests do not hit that line, `SamTemplateQuickPickItem` is masked out because `promptUserForSamTemplate()` is stubbed out in the tests.